src/super_gradients/common/factories/metrics_factory.py

```diff
@@ -0,0 +1,13 @@
+from super_gradients.common.factories.base_factory import BaseFactory
+from super_gradients.training.metrics import Accuracy, Top5, DetectionMetrics
+
+
+class MetricsFactory(BaseFactory):
+
+    def __init__(self):
+        type_dict = {
+            'Accuracy': Accuracy,
+            'Top5': Top5,
+            'DetectionMetrics': DetectionMetrics,
```

Comment:
We have more then these...IoU for example...

Reply:
added IoU and PixelAccuracy